### ISSUE TITLE
Tester Feedback v1.0.preview

### ISSUE THREAD

**Issue by firoball** [Acknex/LotterShock#115]
Ich möchte hier alle Feedback Posts unserer freiwilligen Tester sammeln. Dann können wir uns überlegen, was wir für das gebalancte wirkliche final Release (falls wir noch eins dranhängen) umsetzen.

**Comment by firoball** [comment]
> Habs durch! Also ich find`s ja erstmal mega geil...es wirkt sehr poliert und macht dadurch echt Spaß. Für mich als Neuling, würde ich den Schwierigkeitsgrad (gemessen an Health und Ammo), schon als schwer betiteln. Ich sag mal Stufe 3 von 4. Ich habe vier Versuche gebraucht. Frustrierend fand ich es nicht, aber es war kurz davor. ;D Der Doublejump war ziemlich schwierig. Bin schlecht über die Kanten gekommen. Die Bugs sind vielleicht schon bekannt: wenn man aus dem Spiel tabt, dann wird die Map nicht mehr korrekt angezeigt(keine Map mehr, nur noch der Spielerpfeil). Im gleichen Zug dann die Optionen aufmachen, kam mehrfach ein Spielabbruch "exe funktioniert nicht mehr". Ansonsten macht die Grafik echt Spaß...könnte dazu viel schreiben, aber zusammengefasst-um nicht groß zu spoilern-, finde ich die Detail-liebe echt Spitze (wie zb. auf dem angehangenen Bild) Der Easteregg Room ist auch spitze!

**Comment by firoball** [comment]
> Das waren die nostalgischsten 3 Stunden meines Lebens  :D Erstmal vielen Dank für eure harte Arbeit, es hat sich gelohnt! Es macht unheimlich Spaß. Auch ich würde es als schwer betiteln. Hat man nach dem dritten Run raus, was zu tun ist, welche Waffen effektiv gegen gewisse Gegnertypen sind und wie wichtig Health ist, dann gehts ziemlich klar! Ich habe es durch und durch genossen. Die Grafik ist überraschend konsistent. Am Ende hatte ich mich gerade auf einen Boss eingestellt und der Übergang war so episch.. Doch dann leider nichts :( Das habe aber alle Easter-Eggs wieder wett gemacht. Absolut spitze und mega spaßig!!!!!

**Comment by firoball** [comment]
> I tested on wine.
Some text overlap. Some text goes out of the screen. I wasn't able to make fullscreen work properly. I wasn't able to change resolution. My monitor fits best for 1280x1024. I do not found it too difficult.

This in the photo that say that he is not the real lotti is JCL?

**Comment by MasterQ32** [comment]
> Spiel startet, Intro läuft durch, Menüs funktionieren, aber sobald Spiele-Level geladen wird:
> ![buggy](https://user-images.githubusercontent.com/3932972/43930045-aa2c5e76-9c37-11e8-8191-4e1ee26f7de4.gif)
> [acklog.bug.txt](https://github.com/Acknex/LotterShock/files/2276102/acklog.bug.txt)

**Comment by quadraxas** [comment]
Hi guys!

![untitled](https://user-images.githubusercontent.com/513247/43969988-9f0d2f0a-9cd4-11e8-80ae-b91e2332d9e4.png)

This dude got stuck in the wall. Also the floating red dude is stuck too, but it's not inside a wall, it faces the player direction and kinda move a little bit in a very restricted area, i shot it it might have got stuck between a dead and not dead state.

Both are not shootable(shooting them does nothing). Getting really close to them does not damage player.

**Comment by firoball** [comment]
Recht viel mehr Feedback scheint nicht mehr nachzukommen.
Ich werde kommendes Wochenende nochmal einen offiziellen Release bauen, welcher dann auch auf IndieDB wandert.

Bitte schaut, dass ihr alle Sachen, die ihr noch fixen wollt, diese Woche reinhaut .

Im Zweifelsfall re-release ich ohne Änderungen.

**Comment by mk1x86** [comment]
Ich glaube, du kannst es so releasen. Einige Dinge sind ärgerlich, aber dann ist es jetzt halt so

**Comment by MasterQ32** [comment]
Seh ich auch so, bin eigentlich ganz zufrieden. Der Bug mit dem blinkenden Fenster ist doof, aber auch nicht sinnvoll zu debuggen

**Comment by firoball** [comment]
Ja, releasen geht so, mache ich dann auch.Ich wollte nur ein Zeitfenster für Kleinigkeiten freigeben, wie z.b. fehlendes PDA Sample oder Level einblocken, oder wasauchimmer. Also legt los 😎Am 13.08.2018 3:17 nachm. schrieb Felix Queißner <notifications@github.com>:Seh ich auch so, bin eigentlich ganz zufrieden. Der Bug mit dem blinkenden Fenster ist doof, aber auch nicht sinnvoll zu debuggen

**Comment by firoball** [comment]
Die Fleischbrocken haben jetzt noch Splattersounds. Das war wichtiger als alles andere!

**Comment by MasterQ32** [comment]
Ich hab nen Streamer dazu bewegt, mal das Spiel zu spielen:
https://www.twitch.tv/videos/297289574

Was mir da grade massiv auffällt: Er hat die Auflösung geändert und die HUD hat wohl die Positionen der Keycards und ähnlichem nicht aktiviert. Das müsste man wohl noch anfassen

**Comment by firoball** [comment]
Ich hab mir das mal angeschaut. Die Zusammenfassung:

- Das mit den Keycards ist ein Problem in der Releaseversion. Evtl. wird da was nicht resourced.
- Secret Areas sollten idealerweise noch mit einer Einblendung "You found a secret area" quittiert werden, sonst kapieren es die Spieler nicht.
- Beim CBabe Room scheint wohl dringend noch Einblockung notwendig zu sein.
- Die Automap braucht Transparenz
- Nach dem PowerCore Terminal ist der nächste Schritt augenscheinlich nicht klar.

Den Rest würde ich vernachlässigen, es bleibnt halt ein retro AckCon Projekt ;)

**Comment by xwcg** [comment]
Der Cbabe raum hatte eigentlich einblockung, die ist wohl flöten gegangen durch wer auch immer den kleinen gang-abschnitt mit der blume davor geklatscht hat

**Comment by firoball** [comment]
Das hab dann wohl ich vergeigt :D
Der Raumanbschnitt war notwendig, weil man sich nonst wegen dem Morph in die Tür festmurksen konnte und nicht mehr rauskam.
Ich fixe eh grad ein paar Sachen der Liste

**Comment by firoball** [comment]
gefixt:
- keycards
- secret areas
- cbabe room eingeblockt
- automap
- deadlock im waste recycling

**Comment by firoball** [comment]
Die Map an sich löst bei mir keinen Crash aus, aber tatsächlich ist sie bei einem Auflösungswechsel verschwunden. Ich vermute irgendwas wegen Shader und D3D Device lost? Wer kennst sich da aus und mag das noch schnell fixen?

**Comment by MasterQ32** [comment]
> Die Map an sich löst bei mir keinen Crash aus, aber tatsächlich ist sie bei einem Auflösungswechsel verschwunden. Ich vermute irgendwas wegen Shader und D3D Device lost? Wer kennst sich da aus und mag das noch schnell fixen?

Ich hab da schon zwei, drei Stunden investiert und habs nicht auf die Reihe bekommen, genauso wie die Map mit Alpha zu zeichnen (`draw_quad` ignoriert einfach den Alpha-Wert bei mir).
Kannst es gerne nochmal probieren, aber ich seh da keine Hoffnung... Was **vielleicht** noch tun könnte: Man muss die Bitmap aus dem "Render Target"-Zustand holen, bevor man die Auflösung wechselt

**Comment by firoball** [comment]
also alpha hab ich schon drin.
map_open () scheint irgendwie zu helfen bei Auflösungswechsel, aber da fliegt mir dann beim Starten erstmal die Funktion um die Ohren.
Auf gut deutsch: er verliert wohl den Inhalt der dynamisch erzeugten Bitmaps.

**Comment by firoball** [comment]
also er zeigt dann natürlich komisches Zeug an, weil man ihm die Mapdata kaputt macht und er irgendeine Position der vorangegangenen Auflösung zu nutzen scheint, aber zumindest ist dann wieder was zu sehen.

**Comment by firoball** [comment]
@MasterQ32 dein Tip war wohl goldrichtig. Ein bmap_to_format() vor dem Auflösungswechsel scheint zu helfen

**Comment by firoball** [comment]
Der letzte Teil vom Level ist noch nicht eingeblockt, scheint aber keine Probleme zu haben bislang.
Spielt doch bitte alle nochmal ne Runde zur Probe.
Ich möchte im cbabe room noch ein paar kleine easter eggs verstecken und dann hau ich den Release build endlich raus - außer es gibt berechtigte Einwände natürlich

**Comment by firoball** [comment]
Release ist raus. 
https://github.com/Acknex/LotterShock/releases/tag/v1.0.final
Einen Bug nach dem kompletten Beenden des Spiels und anschließendem Wechsel ins Beastiary konnte ich nicht mehr finden (Cannot Create D3D... schlagmichtot - kommt nur im Release, ich hab so die Vermutung es könnte mit der Hires Atlas Map zu tun haben). Wenn das Problem jemand fixen kann, release ich gerne nochmal.
Und die Sputniks gimpen auch noch immer rum, aber ich glaube sie laufen nicht mehr quer durch irgendwelche Räume und bringen dann Ninja Attacken.

Ansonsten verteilt das Ding, wer Lust hat darf gerne die IndieDB ergänzen, die Homepage erweiter ich dieses Wochenende noch.

**Comment by mk1x86** [comment]
Great job. Danke für dein Durchhaltevermögen

**Comment by firoball** [comment]
Das war eine gemeinsame Arbeit von uns allen :)
Ich hab nur die Reste noch zusammengeräumt.
